### DISCUSSION TITLE
Mom, it still no worky (NPCs fail to start bangin', even when aroused near nodes)

### DISCUSSION THREAD

**Discussion by IndigoCactus** [ErinaSugino/Starbound-SBR-Mod#76] [Trouble Shooting] [answered]
Sex via toys, Starbound Defeat, and Monster Hormones all work fine (mostly) but the basic, NPC horniness-driven sex does not seem to work. Tried in both Outpost and Lustia. NPCs will get together and hearts will fly, but then they'll just stand around until they get bored (or are spoken to) and then go their separate ways.

I should mention I don't have the "Lustbound SBR patch" mentioned in the troubleshooting guide. I could not find it, and assumed that it referred to some outdated mod from before Lustbound was integrated into SBR. If I'm wrong, and said mod exists, please point me to it.

[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/13624910/starbound.log)

**Comment by ErinaSugino** [comment, marked as answer]
Your assumption is correct, the Lustbound SBR patch is not necessary anymore. If I forgot to edit that one out of the troubleshooting guide I'll have to fix that.

Now, as for the hearts you mentioned. That got nothing to do with Sexbound. NPCs having random behavioral reactions to other NPCs is completely vanilla behavior, and solely cosmetic. NPCs don't show heart effects when they are aroused, and they also don't (yet) group together when they are.

Going through your log, there is also not a single error regarding Sexbound in there. So I'm just going to assume the only issue here is impatience. NPCs gain arousal at a default rate that takes 3-7 in-game days, only counting time where they are actively loaded in, to reach 100%. Only then do they go and seek out a free sexnode. I have a vague memory in my mind that something about Lustbound tripled that value, though I can't remember if it was for the NPCs in Lustia or the hookers you can phone in with the Lustbound: Omnibrowser Addon.

**Reply by IndigoCactus** [reply]
Looks to be I have a poor sense of time. Sorry 'bout that, and thanks for the response.
